src/lib/components/Editor/GeneralEditor.tsx

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable react/display-name */
+import React, { useState } from 'react';
+import ReactQuill from 'react-quill';
+import { modules, formats } from './utilities';
+import 'react-quill/dist/quill.snow.css';
+
+export interface IQuillEditorProps {
+  toolbar: React.ReactNode;
+}
+
+const QuillEditorComp = React.forwardRef<ReactQuill | null, IQuillEditorProps>(
```

Comment:
it's good to have the same component name as the file itself.
please use full `component` part of the name :)

Reply:
🤨 
Accidentally missed, I'll update it in the next PR )